README.md

```diff
@@ -81,6 +87,8 @@ npm run dev
 
```

Comment:
Could we also move up and possibly make it more succinct the ".env" duplication?

This would let anyone come in and tick all the boxes quickly before trying to "NPM RUN DEV" and the debugging that would follow

Reply:
purfect